src/Stateless/Reflection/StateInfo.cs

```diff
@@ -102,14 +103,24 @@ public class StateInfo
         public StateInfo Superstate { get; private set; }
 
         /// <summary>
-        /// Actions that are defined to be exectuted on state-entry.
+        /// Actions that are defined to be executed on state-entry.
+        /// </summary>
+        public IEnumerable<MethodInfo> EntryActions { get; private set; }
+
+        /// <summary>
+        /// Actions that are defined to be executed on activation.
+        /// </summary>
+        public IEnumerable<MethodInfo> ActivateActions { get; private set; }
```

Comment:
@nblumhardt I've updated the access from internal to public for consistency, but don't understand your comment about MethodInfo.   MethodInfo is only used in the Reflection part of the public API.  Are you looking at MethodDescription?